cmd/collector/app/span_processor.go

```diff
@@ -96,13 +96,18 @@ func newSpanProcessor(spanWriter spanstore.Writer, additional []ProcessSpan, opt
 	}
 	boundedQueue := queue.NewBoundedQueue(options.queueSize, droppedItemHandler)
 
+	sanitizers := append([]sanitizer.SanitizeSpan{}, sanitizer.StandardSanitizers...)
```

Comment:
`sanitizer.StandardSanitizers` is a globally accessible slice that can be mutated. What do you think about unexporting `sanitizer.StandardSanitizers` and adding a `sanitizer.GetStandardSanitizers()` func?

Then (I don't think we need to append to an empty slice?):
```
sanitizers := sanitizer.GetStandardSanitizers()
```